Source/Tests/LinuxSyscalls/Syscalls.h

```diff
@@ -206,6 +218,94 @@ class SyscallHandler : public FEXCore::HLE::SyscallHandler {
   #endif
 
   std::unique_ptr<FEX::HLE::MemAllocator> Alloc32Handler{};
+  
+  ///// VMA tracking /////
```

Comment:
What does VMA stand for? AFAICT it's not written out in any of the SMC-related PRs, but doing so in one or two (or three) places would help.

Reply:
It's the name the linux kernel uses for this, "Virtual memory area" (https://www.oreilly.com/library/view/linux-device-drivers/9781785280009/4759692f-43fb-4066-86b2-76a90f0707a2.xhtml)

Reply:
Sprinkled in around in comments